lib/validates_url_format_of.rb

```diff
@@ -6,7 +6,7 @@ module ValidatesUrlFormatOf
     \A
     https?://                                        # http:// or https://
     ([^\s:@]+:[^\s:@]*@)?                            # optional username:pw@
-    ( (xn--)?(\-|[a-z0-9])+([-.][a-z0-9]+)*\.[a-z]{2,6}\.? |  # domain (including Punycode/IDN)...
+    ( (xn--)?(\-|[a-z0-9])+([-.][a-z0-9]+)*\.[a-z]{2,13}\.? |  # domain (including Punycode/IDN)...
```

Comment:
A good question. I expect that they'll just continue to add more TLDs, so we probably should. I'd rather not leave it open-ended, but it may be the best way.